cmd/clusterctl/client/cluster/mover_test.go

```diff
@@ -788,3 +788,86 @@ func Test_objectsMoverService_checkTargetProviders(t *testing.T) {
 		})
 	}
 }
+
+func Test_objectMoverService_ensureNamespace(t *testing.T) {
+	type fields struct {
+		fromProxy Proxy
+	}
+	type args struct {
+		toProxy   Proxy
+		namespace string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
```

Comment:
If I'm not mistaken this can be removed and use `fromProxy: test.NewFakeProxy()` at L836